# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/ScrollableLayerGuid.h
# Commit: 0c25260c4edf
# Full Hash: 0c25260c4edf97fdfef11320c2a91bc0698b4c11
# Author: Alexis Beingessner <a.beingessner@gmail.com>
# Date: 2019-07-25 03:53:31
# Regressor Bug: 1547351
# File Overlap Count: 2
# Description:
#   Bug 1547351 - Automatically set content process render root r=kats
#   
#   This implements the idea of automatically setting a content proc's
#   render root based on the render root enclosing the iframe that
#   points to it. There was a bit of cleanup in here that was a bit
# ==============================================================================

diff -r f96bd2aae39c -r 0c25260c4edf gfx/layers/ScrollableLayerGuid.h
--- a/gfx/layers/ScrollableLayerGuid.h	Wed Jul 24 14:10:56 2019 +0000
+++ b/gfx/layers/ScrollableLayerGuid.h	Wed Jul 24 14:10:56 2019 +0000
@@ -27,6 +27,7 @@
 struct ScrollableLayerGuid {
   // We use IDs to identify frames across processes.
   typedef uint64_t ViewID;
+  typedef nsUint64HashKey ViewIDHashKey;
   static const ViewID NULL_SCROLL_ID;  // This container layer does not scroll.
   static const ViewID START_SCROLL_ID = 2;  // This is the ID that scrolling
                                             // subframes will begin at.